pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -20,13 +20,19 @@ import (
 const (
 	repoName         = "rancher-charts"
 	webhookChartName = "rancher-webhook"
+	webhookImage     = "rancher/rancher-webhook"
 )
 
-func Register(ctx context.Context, wContext *wrangler.Context) error {
+type ImageOverride struct {
+	Registry string
+}
```

Comment:
Do we need a struct for this?

Reply:
The idea was to future proof this for the addition of ImagePullSecrets (for authenticated registries). I'm testing now to see if that's necessary. If it's not, I'll remove this and set it as a string instead.

Reply:
I've removed the struct and instead moved to a string - Authenticated registries appear to be set at a container-runtime/cluster level, so we don't seem to need specific image pull secrets here.